outputs.tf

```
@@ -18,3 +18,13 @@ output "administrator_password" {
   value       = azurerm_mssql_server.this.administrator_login_password
   sensitive   = true
 }
+
+output "server_security_alert_policy_id" {
+  description = "The ID of the server security alert policy."
+  value       = azurerm_mssql_server_security_alert_policy.this.id
+}
+
+output "server_vulnerability_assessment_id" {
+  description = "The ID of the server vulnerability assessment."
+  value       = azurerm_mssql_server_vulnerability_assessment.this.id
+}
```

Update descriptions to be more similar to already existing outputs:

```suggestion
output "server_security_alert_policy_id" {
  description = "The ID of this SQL server security alert policy."
  value       = azurerm_mssql_server_security_alert_policy.this.id
}

output "server_vulnerability_assessment_id" {
  description = "The ID of this SQL server vulnerability assessment."
  value       = azurerm_mssql_server_vulnerability_assessment.this.id
}
```